### DISCUSSION TITLE
DTU openDTU on Battery schaltet bei höherer Leistung WR aus

### DISCUSSION THREAD

**Discussion by Agerter** [hoylabs/OpenDTU-OnBattery#759] [Q&A]
Hallo ich habe folgendes Problem das die DTU mit der Nulleinspeisung bei bewölkten Himmel funktioniert aber bei voller Leistung diese den Wechselrichter abschaltet .
Ist dieses Problem vielleicht jemanden Bekannt und kann mir sagen was ich anders einstellen muss.
Vielen Dank

**Comment by spcqike** [comment]
Hallo

nein, dieses Problem ist, zumindest mir, nicht bekannt und sollte auch nicht auftauchen.

Was genau hast du denn eingestellt? wie ist dein Setup? Welche Version der openDTU-onBattery läuft?

Kannst du im DPL die ausführliche Protokollierung einschalten und aus der Konsole einen Log-Auszug herausnehmen, der so ab 1 Minute vor dem Abschalten anfängt?

**Comment by Ralph161065** [comment]
Kenne dieses Problem auch nicht.
Habe mittlerweile zwei Anlagen und gestern viel Sonne.
Dürfte ein "hausgemachtes" Problem sein!

**Comment by SW-Niko** [comment]
Kenne dieses Problem auch nicht.
Bist du sicher das die DTU den Inverter abschaltet? Vielleicht macht das der Inverter eigenständig, wegen Unterspannung oder Überhitzung?
Wie @spcqike schon schreibt, wir benötigen ein paar Daten wie Setup / Protokoll, ansonsten fischen wir nur im trüben und können dir nicht wirklich helfen.
